### PR TITLE
Add korean translation

### PR DESCRIPTION
**Pull request by disjukr**

### CODE REVIEW

File: locales/ko.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "Missing required argument: %s": {
+    "one": "필수 인자를 받지 못했습니다: %s",
+    "other": "필수 인자들을 받지 못했습니다: %s"
+  },
+  "Unknown argument: %s": {
+    "one": "알 수 없는 인자입니다: %s",
+    "other": "알 수 없는 인자들입니다: %s"
+  },
+  "Invalid values:": "잘못된 값입니다:",
+  "Argument: %s, Given: %s, Choices: %s": "인자: $s, 입력받은 값: %s, 선택지: %s",
```
Comment (maxrimue) [inline locales/ko.json]: You mixed in a `$s` there

Comment (disjukr) [inline locales/ko.json]: oh; thanks

Comment (disjukr) [inline locales/ko.json]: fixed

**Comment by sairion** [comment]
Looks good!

**Comment by bcoe** [comment]
Thanks for this contribution, I'm excited that we've gotten so many translations for yargs at this point \o/

I'm releasing `yargs@3.30.0` today, I will work on getting this and #297 released in `3.31.0` this weekend too.

**Comment by disjukr** [comment]
yay \o/

**Comment by bcoe** [comment]
@disjukr could you update https://github.com/bcoe/yargs#localelocale with the new translation too :+1:

**Comment by disjukr** [comment]
done :+1:

**Comment by disjukr** [comment]
oh; i misunderstood the meaning of `count` (https://github.com/bcoe/yargs#yargs-even-counts-your-booleans)
i thought it means `number`. like `--count=123`
`단계`(level) may better than `숫자`(number)

**Pull request merged**